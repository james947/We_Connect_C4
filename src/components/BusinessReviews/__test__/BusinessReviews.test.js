import { shallow } from 'enzyme'
import React from 'react';
import  {ReviewsDisplay}   from '../';

describe("<ReviewsDisplay/>", () => {
    it('renders correctly', () => {
        const reviews = [];
        const wrapper = shallow(<ReviewsDisplay state={ reviews } />);
        expect(wrapper).toMatchSnapshot();
    })
})