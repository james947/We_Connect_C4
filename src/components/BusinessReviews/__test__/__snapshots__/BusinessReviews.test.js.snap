// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReviewsDisplay/> renders correctly 1`] = `
<div>
  <div
    className="card"
  >
    <Connect(withRouter(ReviewPage)) />
    No reviews for this business
  </div>
</div>
`;
